policyengine_us/parameters/gov/states/ks/tax/income/deductions/standard/base_amount.yaml

```
@@ -0,0 +1,23 @@
+description: Base amount a KS income tax unit can get as a standard deduction.
+metadata:
+  label: KS base standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2021 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip21.pdf
+  - title: 2022 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip22.pdf
+
+JOINT:
+  2021-01-01: 8_000
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 6_000
+WIDOW:
+  2021-01-01: 6_000
```

I see that Kansas treats widow(er)s with children as head of household, and doesn't otherwise specify widow(er) filing status values. It's also not in the law. I'd expect that they'd get the joint value.
```suggestion
  2013-01-01: 7_500
  2021-01-01: 8_000
```

---

Done